src/libstd/sys/common/util.rs

```diff
@@ -10,7 +10,6 @@
 
 use env;
 use fmt;
-use intrinsics;
```

Comment:
This will break the `#[cfg(not(unix))]` form below, which still uses `intrinsics`.